### ISSUE TITLE
Build error when trying to run our app with this package

### ISSUE THREAD

**Issue by eallegrucci** [hjkcai/swc-plugin-add-display-name#8]
```
ppo/admin: Error: failed to handle: failed to invoke plugin: failed to invoke plugin on 'Some("/home/runner/work/revvin-frontend-v2/revvin-frontend-v2/apps/admin/src/index.tsx")'
@mortgagehippo/admin: Caused by:
@mortgagehippo/admin:     0: failed to invoke `swc-plugin-add-display-name` as js transform plugin at swc-plugin-add-display-name
@mortgagehippo/admin:     1: RuntimeError: unreachable
```

I think this issue might be similar to this closed issue: https://github.com/hjkcai/swc-plugin-add-display-name/issues/2

**Comment by hjkcai** [comment]
I think the most possible problem is swc has breaking changes in recent versions.

Could you please try the swc version exactly the same as this plugin? `"@swc/core": "1.3.81"`.

If it is, I will upgrade all the dependencies and publish a new version.

**Comment by emyk** [comment]
Yep, must be some breaking changes in swc. We had it working with `"@swc/core": "1.3.102",`, getting the same error after bumping to latest (`"1.4.2"`)

**Comment by hjkcai** [comment]
The plugin code won't compile if I simply update the dependencies. There are More Breaking changes than I am expected. I need some more investigations into swc's source code.

**Comment by hjkcai** [comment]
I have published version `0.4.0` of this plugin. Please help me test if it works. Thanks!
